ansible_collections/arista/avd/roles/eos_designs/python_modules/interface_descriptions/__init__.py

```diff
@@ -0,0 +1,165 @@
+
+import importlib
+from collections import ChainMap
+from functools import cached_property
```

Comment:
mayb we can start enforcing some ordering on the imports between similar classes to enforce consistency? (that's cosmetic)

Reply:
I am actually trying to do that by setting full imports first, standard libraries next and then the internal stuff

Reply:
submitted another PR with isort